google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/CollectionReference.java

```diff
@@ -171,13 +170,13 @@ public void remove() {
    * Adds a new document to this collection with the specified data, assigning it a document ID
    * automatically.
    *
-   * @param fields A Map containing the data for the new document.
+   * @param fields A Map or a POJO containing the data for the new document.
```

Comment:
nit: "a map or a POJO" and no period at end of comment since it's not a complete sentence